drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -560,6 +560,20 @@ def ce_opportunity(id:)
       Hmis::Ce::Opportunity.viewable_by(current_user).find_by(id: id)
     end
 
+    field :ce_candidate, HmisSchema::CeCandidate, null: true do
+      argument :id, ID, required: true
+    end
+
+    def ce_candidate(id:)
+      raise unless Hmis::Ce.configuration.enabled?
+
+      # Candidates aren't tied to a specific project, so check if the user can_view_prioritized_client_lists in any project.
```

Comment:
To discuss - Is this enough for permission guarding this method? 

Do we need to protect against data leakage across data source? I think no, because the client will be a destination client.

Reply:
So, this implementation is saying: if CE is enabled, and the user has "can view prioritized client lists" in any project, then they should be able to resolve _any_ candidate across the system. That doesn't seem right to me, since we said explicitly that the "can view prioritized client lists" permission is scoped to project. (i.e. it was only supposed to grant access to viewing waitlists within the specified project)